### ISSUE TITLE
Docs: Document that !important is ignored in keyframes

### ISSUE THREAD

**Issue by thecalamiity** [eslint/css#146]
### Rule details

Disallow invalid !important within keyframe declarations

### What type of rule is this?

Warns about a potential problem

### Example code

```css
@keyframes foo {
    from {
      opacity: 0;
    }
    to {
      opacity: 1 !important;
    }
}
```

### Participation

- [x] I am willing to submit a pull request to implement this rule.

### Additional comments

Declarations in a keyframe qualified with !important are ignored.

Corresponding stylelint rule - https://stylelint.io/user-guide/rules/keyframe-declaration-no-important/

**Comment by Tanujkanti4441** [comment]
Hi @thecalamiity, thanks for the issue.

I think this case is already reported by `no-important` rule.

**Comment by nzakas** [comment]
@thecalamiity thanks for the suggestion. We don't create rules that overlap with other rules because it becomes difficult for users to understand what's going on. Stylelint's rules tend to be very narrow in focus, often just targeting specific syntax for the same type of error. The ESLint approach is bundle all related violations into a single rule.

It would be a good idea to update the docs for `no-important` to mention this explicitly.

**Comment by snitin315** [comment]
Yes, accepting this as a documentation update for `no-important`.
